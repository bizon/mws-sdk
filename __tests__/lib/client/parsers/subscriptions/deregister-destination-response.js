const parseXml = require('../../../../../lib/client/parsers')
const parseDeregisterDestinationResponse = require('../../../../../lib/client/parsers/subscriptions/deregister-destination-response')

describe('lib.client.parsers.subscriptions.deregister-destination-response', () => {
  it('should parse the DeregisterDestinationResponse example response from MWS doc', () => {
    const doc = parseXml(
      `<?xml version="1.0"?>
      <DeregisterDestinationResponse xmlns="http://mws.amazonservices.com/schema/Subscriptions/2013-07-01">
        <DeregisterDestinationResult />
        <ResponseMetadata>
          <RequestId>b120e610-9cf0-48d7-b615-ca869EXAMPLE</RequestId>
        </ResponseMetadata>
      </DeregisterDestinationResponse>`,
    )

    const response = parseDeregisterDestinationResponse(
      '/subscriptions:DeregisterDestinationResponse',
      doc,
    )

    expect(response).toMatchSnapshot()
  })
})
