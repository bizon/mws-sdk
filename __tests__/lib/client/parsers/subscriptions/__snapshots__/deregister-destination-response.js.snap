// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lib.client.parsers.subscriptions.deregister-destination-response should parse the DeregisterDestinationResponse example response from MWS doc 1`] = `
Object {
  "deregisterDestinationResult": Object {},
  "responseMetadata": Object {
    "requestId": "b120e610-9cf0-48d7-b615-ca869EXAMPLE",
  },
}
`;
